aws/application/application.template

```diff
@@ -534,10 +528,6 @@ Resources:
               Value: !Ref pEnvironment
             - Name: POST_MVP_ENABLED
               Value: !Ref pPostMvpEnabled
-            - Name: PROSECUTION_CONCLUDED_LISTENER_ENABLED
-              Value: !Ref pProsecutionConcludedListenerEnabled
-            - Name: PROSECUTION_CONCLUDED_SCHEDULE_ENABLED
-              Value: !Ref pProsecutionConcludedScheduleEnabled
```

Comment:
Missed reference to `LAA_PROSECUTION_CONCLUDED_QUEUE` environment variable on lines 533 - 534.

Reply:
Removed